smartsim/_core/control/controller.py

```diff
@@ -739,13 +812,71 @@ def _set_dbobjects(self, manifest: Manifest) -> None:
                             if db_script not in ensemble.db_scripts:
                                 set_script(db_script, client)
 
+    def start_telemetry_monitor(self, exp_dir: str) -> None:
+        logger.debug("Starting telemetry monitor process")
```

Comment:
See above comment, about moving the check on `CONFIG.telemetry_enabled` one level up the stack. At the least, this debug line should be moved down, otherwise an erroneous message will be admitted to the debug log

Reply:
moved the `if CONFIG.telemetry_enabled` check